app/helpers/patternfly_components_helper.rb

```diff
@@ -41,6 +42,26 @@ def pf_inline_alert(title, **options)
     end
   end
 
+  # Generates the same HTML as app/javascript/src/utilities/toast.ts#createAlertGroupItem.
+  # Duplication is not ideal, but justified since this will be used by rails template for
+  # flashes set by controllers.
+  def pf_toast_alert(title, **options)
+    action_tag = tag.div class: 'pf-c-alert__action' do
+      tag.button class: 'pf-c-button pf-m-plain', type: 'button', title: 'Close alert' do
+        tag.icon class: 'fas fa-times'
+      end
+    end
+
+    if (type = options[type])
```

Comment:
```suggestion
    if (type = options[:type])
```

Otherwise, the types of alerts don't work and the default is shown always.

Reply:
Good catch!